src/ui/public/vis/vis_types/vislib_vis_type.js

```diff
@@ -76,7 +76,7 @@ export function VislibVisTypeProvider(Private, $rootScope, $timeout, $compile) {
           $scope.vis = this.vis;
           $scope.visData = esResponse;
           $scope.uiState = $scope.vis.getUiState();
-          const legendHtml = $compile('<visualize-legend></visualize-legend>')($scope);
```

Comment:
I like the amount of scope bindings we have for this ;)

Reply:
they really wouldn't make sense here, we are creating a scope just for this directive above.

Reply:
Yeah wasn't meant as a suggestion, but rather as a general joy about the state of our old Angular code :D